addon/models/preprint.js

```diff
@@ -36,6 +36,16 @@ export default OsfModel.extend(ContributorMixin, {
     description: DS.attr('fixstring'),
     tags: DS.attr(),
     public: DS.attr('boolean'),
+    /* Sloan fields */
+    hasCoi: DS.attr('boolean'),
+    hasDataLinks: DS.attr('boolean'),
+    hasPreregLinks: DS.attr('boolean'),
+    dataLinks: DS.attr(),
+    preregLinkInfo: DS.attr(),
```

Comment:
What is the type of the link info?

Reply:
Should be `string`. (`fixstring` is not necessary here because this is a choice field and will never contain special characters)